src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/integration/SqsIntegrationTestBase.kt

```diff
@@ -49,6 +48,7 @@ class SqsIntegrationTestBase : IntegrationTestBase() {
   companion object {
     private val localStackContainer = LocalStackContainer.instance
 
+    @Suppress("unused")
     @JvmStatic
     @DynamicPropertySource
     fun testcontainers(registry: DynamicPropertyRegistry) {
```

Comment:
maybe this entire companion object can be deleted? tests still pass without it and i can’t tell what it’s supposed to do